packages/shared/src/telemetry/events/method-called.ts

```diff
@@ -1,6 +1,7 @@
 import type { TelemetryEventRaw } from '@clerk/types';
 
 const EVENT_METHOD_CALLED = 'METHOD_CALLED';
+const EVENT_SAMPLING_RATE = 0.1;
```

Comment:
Expo environments don't currently throttle based on already-sent events in the session, as we rely on `localStorage` for that. Reducing the sampling rate for now to mitigate excessive events.